src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
```

Comment:
The class `Rectangle` implements `Figure`, but it's not clear if `Figure` is an interface or an abstract class. According to the checklist, you should not use abstract classes to set behavior for classes. Make sure `Figure` is an interface, and if it's currently an abstract class, consider changing it to an interface.